ci_framework/roles/libvirt_manager/tasks/virsh_checks.yml

```diff
@@ -14,47 +14,67 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Start and enable libvirt service
-  become: true
-  ansible.builtin.service:
-    name: "{{ cifmw_libvirt_manager_daemon }}"
-    state: started
-    enabled: true
 
-# stabilize libvirt accesses
-# We hit a fair amount of failures with the libvirt sockets over the
-# past iteration. After many tests, it seems that removing the timeout
-# of the virtproxyd service and ensuring it's enabled and started
-# corrects this issue. With this hack, we could successfully deploy the
-# whole VA-1 layout without any issue.
-- name: Hack in the virtproxyd service
+- name: Remove service override
   become: true
-  when: cifmw_libvirt_manager_apply_virtproxy_patch | bool
   block:
     - name: Create service override directory
       ansible.builtin.file:
         path: "/etc/systemd/system/virtproxyd.service.d"
-        state: directory
+        state: absent
```

Comment:
If we are removing the dir:

- Task name makes no sense: Create service override directory
- Are owner and permissions needed?

Reply:
right - corrected that.